brave/brave-core
# Show default browser dialog instead of infobar

fix https://github.com/brave/brave-browser/issues/12203


<img width="1010" alt="Screen Shot 2021-01-29 at 1 55 34 AM" src="https://user-images.githubusercontent.com/6786187/106171729-40eaf380-61d5-11eb-9178-482707353f29.png">
<img width="1006" alt="Screen Shot 2021-01-29 at 1 56 08 AM" src="https://user-images.githubusercontent.com/6786187/106171841-6972ed80-61d5-11eb-93df-f0d65853aeff.png">


## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.

## Test Plan:
1. Launch browser with any profile (clean or existing both are fine)
2. Repeat launch and check default browser dialog is launched 2nd, 3rd, 4th, 8th and 20th launching
3. Set as default
4. Launch browser and check dialog is launched anymore
5. Set another browser as a default browser
6. Repeat launch and check default browser dialog is launched 2nd, 3rd, 4th, 8th and 20th launching

**Note** If active tab is welcome page, dialog is not visible even if it's 2nd, 3rd, 4th, 8th or 20th launching.
